Source/WebCore/dom/Element.cpp

```diff
@@ -734,7 +734,10 @@ AtomString Element::getAttributeForBindings(const QualifiedName& name, ResolveUR
     }
 
     ASSERT_NOT_REACHED();
+
+IGNORE_RETURN_TYPE_WARNINGS_BEGIN
```

Comment:
Couldn't we solve this without the need for such macros? For example:
```
case ResolveURLs::No:
    break;
}
return attribute->value();
```

I'd really like it if we avoid reduce the spread of such macros to silence warnings.

Reply:
Well it's usually not that easy, or not so nice... but in this case, yes, sure.